src/Workleap.Extensions.OpenAPI/TypedResult/ExtractSchemaTypeResultFilter.cs

```diff
@@ -121,6 +135,28 @@ internal static IEnumerable<ResponseMetadata> GetResponsesMetadata(Type returnTy
         return null;
     }
 
+    private static HashSet<int> ExtractResponseCodesFromAttributes(OperationFilterContext context)
+    {
+        HashSet<int> responseCodes = [];
+        foreach (var attribute in context.MethodInfo.CustomAttributes)
+        {
+            if (attribute.AttributeType.FullName is not (ProducesResponseTypeAttribute or SwaggerResponseTypeAttribute))
```

Comment:
Why not use real Type comparaison? We are not in an Analyzer context here.

And probably there is something to do with just using ProducesResponseTypeAttribute since Swagger inherit from the others one.

Reply:
yup, could do that for sure, something along these lines here:             
`if (!typeof(ProducesResponseTypeAttribute).IsAssignableFrom(attribute.AttributeType))`